src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: $star-size;
```

Comment:
The variable `$star-size` is used here but not defined. Please ensure that this variable is defined in a `vars.scss` file or imported correctly to avoid compilation errors.